acceptance-tests/apps/postgresqlapp/internal/app/app.go

```diff
@@ -17,21 +17,28 @@ const (
 	valueColumn = "valuedata"
 )
 
-func App(uri string) *mux.Router {
+func App(uri string) http.Handler {
 	db, err := connect(uri)
 	if err != nil {
 		log.Fatal(err)
 	}
 	log.Printf("Connection succeeded")
 
-	r := mux.NewRouter()
-	r.HandleFunc("/", aliveness).Methods(http.MethodHead, http.MethodGet)
-	r.HandleFunc("/{schema}", handleCreateSchema(db)).Methods(http.MethodPut)
-	r.HandleFunc("/{schema}", handleDropSchema(db)).Methods(http.MethodDelete)
-	r.HandleFunc("/{schema}/{key}", handleSet(db)).Methods(http.MethodPut)
-	r.HandleFunc("/{schema}/{key}", handleGet(db)).Methods(http.MethodGet)
-	r.HandleFunc("/schemas/{schema}/{table}", handleAlterTable(db)).Methods(http.MethodPut)
-	r.HandleFunc("/", handleDeleteTestTable(db)).Methods(http.MethodDelete)
+	r := chi.NewRouter()
+	r.Head("/", aliveness)
+	r.Put("/{schema}", handleCreateSchema(db))
+	r.Delete("/{schema}", handleDropSchema(db))
+
+	// Although the URL path implies that these might do something in the schema, in fact
```

Comment:
**question**: We haven't updated the routes when we decided to simplify the app and drop the schema requirement on GCP. Schema is used on other platforms. We could use this opportunity to tidy up the routes.

Reply:
I agree this needs doing, but I wanted this PR to be about swapping gorilla/mux to chi. If we add in other things then it's less useful as a reference.